docs/content/app-protect-dos/configuration.md

```diff
@@ -9,7 +9,7 @@ docs: "DOCS-580"
 ---
 
 This document describes how to configure the NGINX App Protect DoS module
-> Check out the complete [NGINX Ingress Controller with App Protect DoS example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/appprotect-dos).
+> Check out the complete [NGINX Ingress Controller with App Protect DoS example for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS example for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/ingress-resources/app-protect-dos).
```

Comment:
I don't think the links with release tag will work after this branch is merged. Are we okay for all the links in the main branch to break until we release v2.4.0?

Reply:
It's ok from a docs standpoint, since we publish from the release branch. 

Should these be pre-seeded with the tags for 2.4.0, since the resources don't exist at these locations in the 2.3.0 branch anyway?

Reply:
I think it's fine to merge to `main` since these docs won't be used until the next release and people don't (shouldn't?) navigate this folder anyway. We usually update the tag during release prep, so it will all be taken care of at that time, I don't see the need to do it right now imo

Reply:
We update all the tag versions in a script as part of the release PR - it will all line up before the 2.4.0 release. Thanks for checking though 😄